### PR TITLE
Created the folder containing files for mask detection

### PR DESCRIPTION
**Pull request by Dharun235**
Added the dataset from kaggle, saved the model file as seperate, added readme file for installation and procedures to implement the project, added the python code for model training and model deployment.

### Requesting to submit a pull request to the PyVerse repository.

---

#### Issue Title
**Please enter the title of the issue related to your pull request.**  
*Enter the issue title here.*
Add a new project
- [x] I have provided the issue title.

---

#### Info about the Related Issue
**What's the goal of the project?**  
*Describe the aim of the project.*
The goal is to create a system that can detect whether a person is wearing a face mask or not using your webcam. The project will involve:

Training a Machine Learning Model: Use a pre-trained model (like MobileNetV2) and fine-tune it to classify images of people with and without face masks.
Real-Time Detection: Integrate the model with OpenCV to process video frames from a webcam and detect if a person is wearing a mask in real-time.
User Feedback: Display a message on the screen indicating whether a mask is present or not, and draw bounding boxes around the detected faces.
- [x] I have described the aim of the project.

---

#### Name
**Please mention your name.**  
*Enter your name here.*
Dharunkumar
- [x] I have provided my name.

---

#### GitHub ID
**Please mention your GitHub ID.**  
*Enter your GitHub ID here.*
Dharun235
- [x] I have provided my GitHub ID.

---

#### Email ID
**Please mention your email ID for further communication.**  
*Enter your email ID here.*
dharun.official.01@gmail.com
- [x] I have provided my email ID.

---

#### Identify Yourself
**Mention in which program you are contributing (e.g., WoB, GSSOC, SSOC, SWOC).**  
*Enter your participant role here.*
Contributor under GSSoC'EXTD and Hacktoberfest
- [x] I have mentioned my participant role.

---

#### Closes
**Enter the issue number that will be closed through this PR.**  
*Closes: #issue-number*
#918
- [ ] I have provided the issue number.

---

#### Describe the Add-ons or Changes You've Made
**Give a clear description of what you have added or modified.**  
*Describe your changes here.*
Added the dataset, model file, code for model training and model deployment in python is added along with readme file.
- [x] I have described my changes.

---

#### Type of Change
**Select the type of change:**  
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

---

#### How Has This Been Tested?
**Describe how your changes have been tested.**  
*Describe your testing process here.*
Tested for model accuracy, then tested for camera feed, then model deployment in the camera feed all in my local pc.
- [x] I have described my testing process.

---

#### Checklist
**Please confirm the following:**  
- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by UTSAVS26** [comment]
@Dharun235 does all `.xml` files required or some files will also work?

**Comment by Dharun235** [comment]
@UTSAVS26 some files will also work. Or otherwise, I can provide the kaggle dataset url in readme file mentioning the user to download the dataset from here.

**Comment by UTSAVS26** [comment]
> @UTSAVS26 some files will also work. Or otherwise, I can provide the kaggle dataset url in readme file mentioning the user to download the dataset from here.

Provide the dataset link.

**Comment by Dharun235** [comment]
I have updated and requesting to check for any other corrections

**Review by ruhi47 — CHANGES_REQUESTED** [review]

**Comment by Dharun235** [comment]
hi @ruhi47, is there anything to change
